requirements.txt

```diff
@@ -34,10 +43,12 @@ pytz==2016.10
 redis==2.10.3
 requests==2.13.0
 requests-oauthlib==0.5.0
-restable>=0.1.3
+requirements==0.1
+restable==0.1.3
+singledispatch==3.4.0.3
+six==1.10.0
+static3==0.7.0
+tornado==4.5.3
 Unidecode==0.4.20
 urllib3==1.19.1
 xmltodict==0.10.2
```

Comment:
are all these additional requirements necessary?

Reply:
Man that was fast. The only thing I installed in this ticket was chardet do I assumed that these all belonged with it. I looked at the documentation and didn't see anything about the dependencies so I have removed what I think is unnecessary.